<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Readme.utf8</title>

<script src="site_libs/header-attrs-2.6/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Big Data Biogeography</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="Readme.html">
    <span class="fas fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="01_schedule.html">Course schedule</a>
</li>
<li>
  <a href="02_student_project.html">Student project</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Day 1
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="mo_overview.html">Overview</a>
    </li>
    <li>
      <a href="mo_setup.html">Setup</a>
    </li>
    <li>
      <a href="mo_download_gbif.html">Download from GBIF</a>
    </li>
    <li>
      <a href="mo_download_paleobioDB.html">Download from Paleobiology database</a>
    </li>
    <li>
      <a href="mo_download_iucn.html">Download from IUCN database</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Day 2
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="tue_overview.html">Overview</a>
    </li>
    <li>
      <a href="tue_clean_geographic_data.html">1. Cleaning geographic records from public databases</a>
    </li>
    <li>
      <a href="tue_clean_fossils.html">2. Cleaning fossil data (optional)</a>
    </li>
    <li>
      <a href="tue_sampling_bias.html">3. Quantifying sampling bias</a>
    </li>
    <li>
      <a href="tue_probabilistic_cleaning.html">4. Probabilistic cleaning with bRacatus</a>
    </li>
    <li>
      <a href="tue_species_ranges_and_richness.html">5. Species richness and range size</a>
    </li>
    <li>
      <a href="tue_further_reading">Further reading</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Day 3
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="wed_overview.html">Overview</a>
    </li>
    <li>
      <a href="wed_taxonspecific_bioregionalization.html">1. Taxon-specific bioregionalization</a>
    </li>
    <li>
      <a href="wed_ancestral_areas_DEC.html">2. Reconstructing ancestral areas (DEC)</a>
    </li>
    <li>
      <a href="wed_DEC_bsm.html">3. Estimating the number of shifts (DEC BSM)</a>
    </li>
    <li>
      <a href="wed_diversification_rates_geosse.html">4. Diversification rate estimation (GeoSSE)</a>
    </li>
    <li>
      <a href="wed_further_reading">Further reading</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Day 4
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="thu_overview.html">Overview</a>
    </li>
    <li>
      <a href="thu_aa_criterion_B.html">1. Automated conservation assessments IUCN Criterion B</a>
    </li>
    <li>
      <a href="thu_aa_neural_network.html">2. Automated assessments using Neural Networks</a>
    </li>
    <li>
      <a href="thu_further_reading">Further reading</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Day 5
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="fr_overview.html">Overview</a>
    </li>
    <li>
      <a href="fr_presentations.html">Student presentations</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/azizka/big_data_biogeography">GitHub</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="physalia-courses-big-data-biogeography" class="section level1">
<h1>Physalia courses: Big Data Biogeography</h1>
<p>Date &amp; time: February 1st - February 5th</p>
<p>Times (CET):</p>
<ul>
<li><p>9:00 - 12:00 Exercises (teachers available for questions via slack)</p></li>
<li><p>13:00 - 17:00 Lectures, demonstrations and group work (synchronous)</p></li>
</ul>
<p>Location: on-line</p>
<p>Teachers: Alexander Zizka, German Center for Integrative Biodiversity research &amp; Daniele Silvestro, University of Fribourg, Switzerland</p>
<div id="schedule" class="section level2">
<h2>Schedule</h2>
<p>The course consists of five days with different topics. You can find a detailed schedule in the overview tab for each day.</p>
<ul>
<li><p>Day 1 - Biodiversity databases. Introduction to different types of large-scale biodiversity data and methods for reproducible data retrieval.</p></li>
<li><p>Day 2 - Data quality and processing. Understanding common issues with species distribution data from large-scale biodiversity databases and introduction to methods to address them.</p></li>
<li><p>Day 3 - Historical biogeography</p></li>
<li><p>Day 4 - Big data conservation assessment</p></li>
<li><p>Day 5 - Fossil biogeography</p></li>
</ul>
</div>
<div id="important" class="section level2">
<h2>Important</h2>
<p>We will use Slack for most of the communication before and during the course. Please make sure to check the slack channel regularly.</p>
</div>
<div id="how-the-course-works" class="section level2">
<h2>How the course works</h2>
<ul>
<li><p>We will meet for the first time on Monday 13:00 CET on zoom. Please make sure to have a look at <span class="citation">@before_start</span>, so that we can start swiftly.</p></li>
<li><p>The course will be split into daily live sessions (13:00 - 17:00 CET) and related asynchronous exercise session on which you will work between the live sessions. You can chose the timing of the exercise sessions, and we will be available to answer questions via slack from 9:00 - 12:00 CET every day.</p></li>
<li><p>During the live session we will consist of <em>lectures</em> where we will present theoretical concepts and a broader context for the exercises of each day and <em>demonstrations</em> where we will briefly present the analysis work-flows for each day and you will then have time to explore them and ask questions. During the asynchronous sessions you will independently address specific <em>exercises</em> following tutorials.</p></li>
<li><p>Detailed information on the course are available on the course webpage (<a href="https://azizka.github.io/big_data_biogeography/" class="uri">https://azizka.github.io/big_data_biogeography/</a>) which we will update constantly during the course. On the webpage you will find for each day:</p>
<ul>
<li>the exercises with tutorial</li>
<li>a detailed schedule</li>
<li>learning objectives and expected outcomes</li>
<li>further reading</li>
</ul></li>
<li><p>During the course you will work on your own project, applying the methods presented during the course to your own data. To do so please bring a taxonomic group of interest (ideally up to 200 species, and if possible with a phylogeny available), and think of some questions regarding this dataset that you would like to answer. This will give you the opportunity to chose questions and exercises most suitable for your work and get feedback from the teacher. There will be example data for all exercises, in case you do not have your own data yet. At the end of the course you will briefly present your results. You can find more information on the project on the <a href="https://azizka.github.io/big_data_biogeography/">course webpage</a>.</p></li>
</ul>
</div>
<div id="contact" class="section level2">
<h2>Contact</h2>
<p>Course content: <a href="mailto:alexander.zizka@idiv.de">Alexander Zizka</a> Organisational: <a href="mailto:info@physalia-courses.org">Physalia courses</a></p>
<p><a href="https://www.physalia-courses.org/courses-workshops/course48/">Physalia courses webpage</a></p>
</div>
<div id="objectives" class="section level2">
<h2>Objectives</h2>
<p>After this course, students will be able to:</p>
<ol style="list-style-type: decimal">
<li><p>Obtain and prepare large scale species occurrence records from public databases in R (including data mining, data cleaning and exploration)</p></li>
<li><p>Apply novel methods for handling and processing ‘big data’ in biogeographic research, including area classification, bioregionalization and automated conservation assessments</p></li>
<li><p>Reconstruct species ancestral ranges based on species occurrences and phylogenetic trees, using different evolutionary models</p></li>
<li><p>Understand the potential and caveats of fossil based biogeography, and be familiar with novel methods to estimate ancestral ranges and evolutionary rates from ranges of extinct and extant taxa</p></li>
</ol>
</div>
<div id="background" class="section level2">
<h2>Background</h2>
<p>The public availability of large-scale species distribution data has increased drastically over the last ten years. In particular, due to the aggregation of records from museums and herbaria, and citizen science in public databases such as the Global Biodiversity Information Facility (GBIF). This is leading to a ‘big data’ revolution in biogeography, which holds an enormous but still poorly explored potential for understanding large scale patterns and drivers of biodiversity in space and time.</p>
</div>
<div id="course-literature" class="section level2">
<h2>Course literature</h2>
<ol style="list-style-type: decimal">
<li><p>Meyer et al. (2015) Global priorities for an effective information basis of biodiversity distributions. Nature Communications, 8 pp.</p></li>
<li><p>Antonelli et al. (2018) Amazonia is the primary source of Neotropical biodiversity. PNAS 115(23): 6034–6039.</p></li>
<li><p>One of the following suggestions (depending on your own interests):</p>
<ol style="list-style-type: lower-alpha">
<li><p>Edler et al. (2017) Infomap Bioregions: Interactive mapping of biogeographical regions from species distributions. Systematic Biology 66(2):197–204.</p></li>
<li><p>Zizka et al. (2019) CoordinateCleaner: Standardized cleaning of occurrence records from biological collection databases. Methods in Ecology and Evolution 10:744-751.</p></li>
<li><p>Price et al. (2019) Big data little help in megafauna mysteries. Nature 558(7):23-25</p></li>
</ol></li>
</ol>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
